doc/contributing/issues.md

```diff
@@ -60,5 +60,12 @@ activities, such as applying labels and closing/reopening/assigning issues.
 For more information on the roles and permissions, see ["Permission levels for
 repositories owned by an organization"](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization).
 
+triagers are requested to abstain from below actions:
+* approve or request changes to a pull request
```

Comment:
I don't think it is appropriate to ask triagers abstain from request changes to a pull request. [Any contributors](https://github.com/nodejs/node/blob/main/doc/contributing/pull-requests.md#reviewing-pull-requests) are welcomed to give reviews on a pull request. It's just that only collaborators can effectively block a pull request from landing (but a clear reason is still required anyway).

My feeling is that given the additional permissions as a triager, a triager should be sticking to the project standard, like [the guide](https://github.com/nodejs/node/blob/main/doc/contributing/collaborator-guide.md#closing-issues-and-pull-requests) for collaborators:

1. Welcoming first-time contributors/issue posters
2. Where this is unclear, leave the issue or pull request open for several days to allow for discussion.
3. Most importantly, be friendly.

Reply:
[GH docs](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization#permission-levels-for-repositories-owned-by-an-organization) shows two different actions separately:
 - submit reviews on PRs
 - approve or request change to PRs
 
while anyone is free to review PRs, approval and request change is better reserved to committers, lest it can confuse contributors (especially new comers)?

Reply:
I can see how that'd be true for folks who have write permission on the repo but are not collaborators. I'm not sure I understand how that'd apply to triaggers. For triaggers who are collaborators: why would we be asking them to refrain? For triaggers who not collaborators: why would it different from other non-collaborator contributors?

Reply:
I think the clarification should focus on the new priviledges granted with the role and responsibility of the role. Like in [the collaborator guide](https://github.com/nodejs/node/blob/main/doc/contributing/collaborator-guide.md), the new priviledges are "writing to the repo". And in terms of the triager role, it is "managing the issues".

"Request a change" is not a new priviledge granted as a triger. Anyone, non-collaborators, can request change with the GitHub UI. And GitHub UI distinguishes effectiveness of a change request with red icons or grey icons.

Reply:
that is where the problem I guess. you both raise valid points. however when looking from the author point of view, they could get confused when a request change coming from a triager and perceive it as different (and more enforcing) than that from a normal contributor, while in reality its the same. how do we tell a triager to `not pretend` they have additional privilege while reviewing a PR?

Reply:
If becoming a triagger results in being forbidden to express your opinion, that seems like a plain bad deal.

- Do we have reasons to believe there's a confusion for external contributors? (i.e. that GitHub UI showing the review as grey is not enough)
- If being a triagger gives your reviews more credibility, is that a bad thing? I see it as a perk for contributing to a project.

I guess as a TSC voting member, my reviews/comments may also be perceived as "more important" than collaborator who is not a TSC voting member, yet there's no difference according to our policy. Does that mean I should refrain from expressing my opinion in case someone thinks my opinion is more valuable than it actually is?
I certainly agree that we should do our best to reduce power imbalance, and have process that would be less likely to confuse external contributors, but IMO we should still hear everyone.

Reply:
👋 Non-Collaborator Triager here, and here are my two cents:

I’ve been triaging for this project for about six months. During this time, I generally haven’t been using “approve” or “suggest changes” in my PR reviews. Instead, if I have feedback, I leave it as a “comment” review.

That said, I don’t believe that avoiding “suggest changes” diminishes the impact of my feedback. However, giving a "approve" review can, in my opinion, give users a boost by saying that their code looks good.

So, I think for triagers to have the most _positive_ impact, they should avoid "suggest changes", but not "approve".

Reply:
@gireeshpunathil no matter what I think, the text in the PR should clarify to whom it applies to. i.e. you're the one who should clarify, not me :D

Reply:
for me, this applies to triagers, who are wanting to exercise their triage role.

when they are also a collaborator or a tsc member, their respective role definitions augment into this one, and the highest role among all apply for a specific action. that is neither captured here, nor overridden here.

from your explanation, the `action` pertinent to the role was also not clear to me. `review` looks like an overloaded term. it comprises at least of:
 - submit reviews on PRs
 - approve or request changes on PRs

reference from GH docs (second column is for triagers)
 
<img width="743" alt="Screenshot 2024-11-08 at 8 02 41 PM" src="https://github.com/user-attachments/assets/b419409b-645a-43f2-9852-63ef9e5495de">

I am not proposing triagers refrain from submitting reviews. instead, approving or requesting changes. triagers do not have permission to do those, but if they perform these, it may confuse authors. so make it explicit here.

Reply:
> triagers do not have permission to do those

I'm not sure I agree, anyone can submit a review, approve or request changes. Anyway, I think we do not agree on whether the difference in GH UI is enough to avoid confusion, and we'll have to agree to disagree.

Reply:
By "anyone can submit a review, approve or request changes.", I was meaning "it is a feature that GitHub offers", I didn't mean that anyone can block a PR from landing.

Reply:
given that we agreed to disagree on whether an author will be misled by a triager action on PRs and whether GH UI color code is sufficient to disambiguate that confusion or not, I will yield and go ahead with what we have consensus on.

thanks @aduh95 !

Reply:
(ofc, the other topic: whether anyone can use all PR review actions or not - has wider scope, and just for triagers, and doesn't need to be sorted out here)